generators/server/templates/dotnetcore/src/Project/Project.csproj.ejs

```diff
@@ -70,6 +69,9 @@
         <PackageReference Include="Ocelot"  />
         <PackageReference Include="Ocelot.Provider.Consul"  />
         <%_ } _%>
+        <%_ if(!skipClient && clientFramework === "Blazor") { _%>
```

Comment:
here you don't check the argument include blazor, maybe we can introduce a choice for blazor webassembly or blazor server (include blazor) ?

Reply:
@NicolasRitouet  Here I'm adding the package "Microsoft.AspNetCore.Components.WebAssembly.Server"

If I add the IncludeBlazor condition:
```<PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.Server"  Condition="$(INCLUDE_BLAZOR) == 1" />

Now all the configuration code to allow including blazor (MvcStartup.cs) will not even compile (because the package is missiog).

The cons of this is a small cost of adding the package reference to the project, even if it is not being used. 

But it is a small cost.

I will mention you on a comment at the code that would break

Reply:
I think having that package reference is a very small cost to pay to be able to serve blazor. Otherwise code will not compile without that package

Reply:
@nicolas63  I think having that package reference is a very small cost to pay to be able to serve blazor. Otherwise MvcStartup.cs code will not compile without that package.

That is why I think we should keep that package